docs/appendix/zowe-yaml-configuration.md

```diff
@@ -46,7 +46,32 @@ The Zowe YAML configuration file has template logic for relating one value to an
 It can help to make your configuration portable between systems that need slightly different behavior while retaining the same configuration file.
 More details can be found [in the configmgr documentation.](../user-guide/configmgr-using.md#configuration-templates)
 
-### Configuration override
+### Configuration override - defaults.yaml
+
+There is a [`defaults.yaml`](https://github.com/zowe/zowe-install-packaging/blob/v3.x/staging/files/defaults.yaml) file, which defines values for global configuration and components. This file is always merged with current config(s) when `configmgr` is used.
+
+For example, if you decide to remove `zowe.job` section by commenting or deleting, after the merging with the defaults the `zowe.job` section will be present again:
+
+**Example of user config:**
```

Comment:
Perhaps it would be useful to show the initial configuration prior to the user config.